dotnet/test/Azure.Iot.Operations.Connector.UnitTests/TestMountFiles/ConnectorConfig/AIO_METADATA

```diff
@@ -0,0 +1,4 @@
+﻿{
```

Comment:
can you put these files in the /eng folder - I bet these will be helpful for testing in the other languages too

Reply:
Good idea. Done.